src/Result/src/ResultFactory.cpp

```diff
@@ -19,15 +20,15 @@ namespace calculator
 // ResultFactory public methods /
 ///
    
-    Result ResultFactory::CreateResult(const Expression &parsedExpression, const float &answer)
+    Result ResultFactory::createResult(const Expression &parsedExpression, float answer)
     {
-        Result result;
-        result.expression = parsedExpression;
-        result.answer = answer;
-        std::stringstream ss;
-        ss << parsedExpression.a << CalculatorMessages::EMPTY_SPACE << parsedExpression.operation << CalculatorMessages::EMPTY_SPACE << parsedExpression.b << CalculatorMessages::EQUALS << answer << std::endl;
-        result.fullResult = ss.str();
-        return result;
+        std::stringstream stringStream;
```

Comment:
You don't need a stringstream here. Concatenation with the + operator should work if everything is a string.

It might be clearer if you do.  
```cpp
std::string fullResult = std::to_string(parsedExpression.a) + CalculatorMessages::EMPTY_SPACE +
parsedExpression.operation + CalculatorMessages::EMPTY_SPACE +
std::to_string(parsedExpression.b) + CalculatorMessages::EQUALS +
 to_string(answer) + "\n";
```

This is personal preference. Your current approach is fine, but this will look clearer for people not familiar with C++

Reply:
Ah, that makes sense. 👍